GVRf/Framework/jni/engine/renderer/renderer.cpp

```diff
@@ -74,6 +173,11 @@ void Renderer::renderCamera(Scene* scene, Camera* camera, int framebufferId,
     numberDrawCalls = 0;
     numberTriangles = 0;
 
+    int currentEye = camera->render_mask() - 1;
```

Comment:
This makes assumptions about `render_mask()` - might be better to hide that in a `camera->get_eye()` call.